web-ui/view/github.ml

```diff
@@ -390,19 +390,28 @@ let show_step ~org ~repo ~refs ~hash ~jobs ~variant ~job ~status ~csrf_token
   let line_number = ref 0 in
   let decorate data : string =
     let aux l log_line =
-      line_number := !line_number + 1;
-      Fmt.str "%s\n%s" l
-        (Fmt.to_to_string (pp_elt ())
-           (div
-              ~a:[ a_class [ "code-line" ]; a_id (Fmt.str "L%d" !line_number) ]
-              [
-                div
-                  ~a:[ a_class [ "code-line__number" ] ]
-                  [ txt (Fmt.str "%d" !line_number) ];
-                div
-                  ~a:[ a_class [ "code-line__code" ] ]
-                  [ Unsafe.data log_line ];
-              ]))
+      match log_line with
+      | "" ->
+          l
+          (* Don't include empties.
+             We seem to get these while we wait on the stream *)
+      | log_line ->
+          line_number := !line_number + 1;
+          Fmt.str "%s\n%s" l
+            (Fmt.to_to_string (pp_elt ())
```

Comment:
Why not use an `%a` here as you rely on the formatter to produce the result?

Reply:
That would be more idiomatic use of Fmt